api/dp-openapi.yaml

```diff
@@ -591,6 +591,354 @@ paths:
+          type: string
+      requestBody:
+        description: data for operation
+        required: true
+        content:
+          application/json:
+            schema:
+              oneOf:
+                - $ref: '#/components/schemas/page_save'
+                - $ref: '#/components/schemas/page_return'
+                - $ref: '#/components/schemas/page_mark_bad'
+      responses:
+        200:
+          description: operation success
+          content: {}
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        404:
+          $ref: '#/components/responses/NotFound'
+        422:
+          $ref: '#/components/responses/InvalidCharacters'
+        429:
+          $ref: '#/components/responses/RateLimitExceeded'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+
+  /proofrounds/{proofroundid}/proofprojects/{proofprojectid}/ownedpages/{ownedpage}:
+    put:
+      tags:
+      - project
+      description: make page available for proofreading, change state to temp if necessary
+      parameters:
+      - name: proofroundid
+        in: path
+        description: The round identifier (as defined by array_keys($Round_for_round_id_)
+        required: true
+        schema:
+          type: string
+      - name: proofprojectid
+        in: path
+        description: The project id
+        required: true
+        schema:
+          type: string
+      - name: ownedpage
+        in: path
+        description: the name of the page, must be in temp state of saved state and owned by user
+        required: true
+        schema:
+          type: string
+      responses:
+        200:
+          description: Page data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/proof_page'
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        404:
+          $ref: '#/components/responses/NotFound'
+        429:
+          $ref: '#/components/responses/RateLimitExceeded'
+        default:
+          $ref: '#/components/responses/UnexpectedError'
+
+  /proofrounds/{proofroundid}/proofprojects/{proofprojectid}/proofpages/{proofpage}/pageoptions/{pageoption}:
+    get:
+      tags:
+      - project
+      description: get page text from either current or previous round
+      parameters:
+      - name: proofroundid
+        in: path
+        description: The round identifier (as defined by array_keys($Round_for_round_id_)
+        required: true
+        schema:
+          type: string
+      - name: proofprojectid
+        in: path
+        description: The project id
+        required: true
+        schema:
+          type: string
+      - name: proofpage
+        in: path
+        description: the name of the page, must be in temp state and owned by user
+        required: true
+        schema:
+          type: string
+      - name: pageoption
+        in: path
+        description: current or original
```

Comment:
should we use enum for the scheama? https://swagger.io/docs/specification/data-models/enums/